Rephrase some explanations and add some missing things (#61)
CloudNetService/website: versioned_docs/version-3.4/commands/service.md

id: service
title: Service Command
sidebar_label: Service

> description: Manages all services on the local node and in the cluster

Reviewer: In my opinion
```suggestion
description: Manages all services in the cluster
```
is enough to understand what the command does, but it might be confusing for unexperienced users.

**Usage:** `service <name> includeInclusions`

This command downloads all [inclusions](../components/tasks.md#includes) that have not yet been downloaded.  

> It's useful if new inclusions are added or updated, and they need to be updated for the service

Reviewer: The `and they need to be updated for the service` part is definitely confusing. Also the inclusions were added.

> It's useful if new inclusions are added or updated, and they need to be updated for the service

Reviewer 2: We shouldn't use the short forms in "official texts"

> The command is useful if new inclusions are added or updated, and they need to be updated in the service

Reviewer 3: ```suggestion
It's useful if new inclusions are added or updated, and they need to be updated for the service
```
This phrasing is better in my opinion

## service includeTemplates {#service-includetemplates}

**Usage:** `service <name> includeTemplates`

This command downloads all [templates](../components/tasks.md#templates) that have not yet been downloaded.  

> It's useful if new templates are added or updated, and they need to be updated for the service

Reviewer: Same thing here.

> It's useful if new templates are added or updated, and they need to be updated for the service

Reviewer 2: Same as above

> The command is useful if new templates are added or updated, and they need to be updated in the service

Reviewer 3: ```suggestion
It's useful if new templates are added or updated, and they need to be updated for the service
```
This phrasing is better in my opinion
